src/main/java/helperClasses/SingletonBrowser.java

```diff
@@ -0,0 +1,35 @@
+package helperClasses;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import projectVeriables.ProjectVariables;
+
+import java.time.Duration;
+
+public class SingletonBrowser {
+
+    private static SingletonBrowser instance =null;
+    private static WebDriver driver;
```

Comment:
It wasn't static, why did you make it static?

Reply:
fixed